lib/package/kinesis/compileMethodsToKinesis.js

```diff
@@ -48,13 +48,13 @@ module.exports = {
       'Fn::GetAtt': ['ApigatewayToKinesisRole', 'Arn']
     }
 
+    const integrationUri = this.getIntegrationUri(http)
+
     const integration = {
       IntegrationHttpMethod: 'POST',
       Type: 'AWS',
       Credentials: roleArn,
-      Uri: {
-        'Fn::Sub': 'arn:aws:apigateway:${AWS::Region}:kinesis:action/PutRecord'
-      },
+      Uri: integrationUri,
```

Comment:
Why not just:

```
Uri: this.getIntegrationUri(http),
```

And then you can remove line 51?

Reply:
Refactored accordingly